pkg/utils/o11y/controller.go

```diff
@@ -83,3 +88,68 @@ func (h *SuiteController) GetRegexPodNameWithSize(podNameRegex string, results [
 
 	return podNameWithSize, nil
 }
+
+func (h *SuiteController) QuayImagePushPipelineRun(quayOrg, secret, namespace string) (*v1beta1.PipelineRun, error) {
+	pipelineRun := &v1beta1.PipelineRun{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "pipeline-egress-",
+			Namespace:    namespace,
+		},
+		Spec: v1beta1.PipelineRunSpec{
+			PipelineSpec: &v1beta1.PipelineSpec{
+				Tasks: []v1beta1.PipelineTask{
+					{
+						Name: "buildah-quay",
+						TaskSpec: &v1beta1.EmbeddedTask{
+							TaskSpec: v1beta1.TaskSpec{
+								Steps: []v1beta1.Step{
+									{
+										Name:  "pull-and-push-image",
+										Image: "quay.io/redhat-appstudio/buildah:v1.28",
+										Env: []corev1.EnvVar{
+											{Name: "BUILDAH_FORMAT", Value: "oci"},
+											{Name: "STORAGE_DRIVER", Value: "vfs"},
+										},
+										Script: fmt.Sprintf(`#!/bin/sh
+												// srcImageRef="quay.io/redhat-appstudio/hacbs-test:latest"
```

Comment:
```suggestion
```

Reply:
readded in new pull request. will remove it.

Reply:
Done